pom.xml

```diff
@@ -80,13 +80,13 @@
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
 
         <!-- Maven plugins version -->
-        <maven.compiler.version>3.8.1</maven.compiler.version>
-        <maven.checkstyle.version>3.1.2</maven.checkstyle.version>
-        <maven.gpg.version>3.0.1</maven.gpg.version>
-        <maven.jacoco.version>0.8.7</maven.jacoco.version>
-        <maven.javadoc.version>3.3.1</maven.javadoc.version>
-        <maven.surefire.version>3.0.0-M5</maven.surefire.version>
-        <maven.source.version>3.2.1</maven.source.version>
+        <maven.compiler.version>3.11.0</maven.compiler.version>
+        <maven.checkstyle.version>3.3.1</maven.checkstyle.version>
+        <maven.gpg.version>3.1.0</maven.gpg.version>
+        <maven.jacoco.version>0.8.11</maven.jacoco.version>
+        <maven.javadoc.version>3.6.0</maven.javadoc.version>
+        <maven.surefire.version>3.2.1</maven.surefire.version>
+        <maven.source.version>3.3.1</maven.source.version>
```

Comment:
The last version for maven-source-plugin is the 3.3.0, the version 3.3.1 doesn't exist and provide an error, you can run this command  "mvn source:aggregate" to make sure